script/ping_websites.rb

```diff
@@ -44,46 +44,32 @@ def url_exist(name, url_string)
     if res.kind_of?(Net::HTTPRedirection)
         # Do nothing
     elsif res.code == "404"
-        # Some webpages return 404 even though they properly redirect to login pages
-        STDERR.puts "Entry: #{name} returned #{res.to_s}"
+        # Some pages 404 even though they properly redirect to login pages
```

Comment:
I think we can remove this comment, because after my latest change it doesn't happen anymore

Reply:
I assume that's only because sites.json doesn't have any pages that give HTTP 404 responses? Or am I missing something here?

Reply:
Before c609a24d there were around a dozen websites falling, but they were actually alive. Afterwards, these false positives disappeared

Reply:
Does that make this comment invalid? Seems like it's worth keeping in case other sites end up going through this code path in the future.